tests/Findologic/Request/NavigationRequestFactoryTest.php

```diff
@@ -25,13 +26,15 @@ class NavigationRequestFactoryTest extends TestCase
      */
     public function testNavigationRequestInstance(): void
     {
+        $_SERVER['HTTP_CLIENT_IP'] = '192.168.0.1';
```

Comment:
Why was this added? How was it working before? Because this is an existing test, and we are also checking for expected IP address, so why was this needed to be added?

Reply:
While executing there is an error showing 'invalid parameter userip' so I checked that this array was not set, that's why I assigned it manually.

Reply:
Yes I understand that, but why was this change needed in this story? It does not seem relevant to the story and it is an existing test so it should be passing already without this change.

Reply:
I'll have look on it once again, I know it shouldn't give an error but it was like that,

Reply:
![error](https://user-images.githubusercontent.com/38109482/144121099-fed0981f-5e4e-4cd4-8dd7-5a07ee4230e5.jpg)
While I am running a test, I am having this error after removing this line

Reply:
I have removed this line because this was working fine on the git test but not locally.